src/modules/Common/public/Start-SdnDataCollection.ps1

```diff
@@ -224,6 +225,20 @@ function Start-SdnDataCollection {
                 }
             }
 
+            # check to see if any network traces were captured on the data nodes previously
+            "Checking for any previous network traces and moving them into {0}" -f $tempDirectory.FullName | Trace-Output
+            Invoke-PSRemoteCommand -ComputerName $dataNodes -ScriptBlock {
+                [System.IO.FileInfo]$networkTraceDir = "$($using:workingDirectory)\NetworkTraces"
+                if (Test-Path -Path $networkTraceDir.FullName -PathType Container) {
+                    try {
+                        Move-Item -Path "$($using:workingDirectory.FullName)\NetworkTraces" -Destination $using:tempDirectory.FullName -ErrorAction Stop
```

Comment:
Add one more commit to add test case for start/Stop-SdnNetshTrace. Hit below if Temp folder didn't cleanup and NetworkTraces folder existed:
```
WARNING: Unable to move C:\Windows\Tracing\SdnDataCollection\NetworkTraces to C:\Windows\Tracing\SdnDataCollection\Temp
 System.IO.IOException: Cannot create a file when that file already exists.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.DirectoryInfo.MoveTo(String destDirName)
   at Microsoft.PowerShell.Commands.FileSystemProvider.MoveDirectoryInfoItem(DirectoryInfo directory, String 
destination, Boolean force)
```

This won't happen if Temp folder always cleaned up before data collection.  Please confirm if -Force needed or not to overwrite any existing file.

Reply:
Typically, the \Temp directory will be cleaned up as part of the start and end of `Start-SdnDataCollection` function so we should be fine here, however I did add a `-Force` switch which should also prevent that.